### DISCUSSION TITLE
yiwen-blog-website/blog/2024_0722

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#267] [Announcements]
# yiwen-blog-website/blog/2024_0722

今天和往常相同，但是也有一些不同，因为仿今日头条app终于开发完了。我也终于可以搞ai了。终于要进入ai领域了，相较于其他领域的红海，这里可谓是大有前途。

https://hao-yiwen.github.io/yiwen-blog-website/blog/2024_0722

**Comment by Hao-yiwen** [comment]
2024年0818日，我的电动车坏了。电动车陪我度过了事业中的重要关口，希望能修好电动车，没有电动车，我该怎么活啊。呜呜呜~~~

**Reply by Hao-yiwen** [reply]
经过师傅排查 发现是电动车充电器坏了 结果还能接受。。
